class Base_Exception(Exception):
    pass